tests/e2e/api/fungible_asset.test.ts

```diff
@@ -19,4 +19,23 @@ describe("FungibleAsset", () => {
     expect(data[0]).toHaveProperty("asset_type");
     expect(data[0].asset_type).toEqual(APT_COIN_TYPE);
   });
+
+  test("it should fetch fungible asset activities with correct number and asset type ", async () => {
+    const config = new AptosConfig({ network: Network.LOCAL });
```

Comment:
let's declare all those repated variables (config, aptos, APT_COIN_TYPE, etc) at the top of the file

Reply:
Good call. I will address this part in the next PR on the stack, so there won't be conflict.